src/main/java/org/sonatype/goodies/packageurl/PackageUrl.java

```diff
@@ -75,20 +72,20 @@
   /**
    * Values should have already been validated via {@link PackageUrlBuilder} and {@link PackageUrlValidator}.
    */
-  @VisibleForTesting
+  // @VisibleForTesting
```

Comment:
Yeah, I see what you are saying.

But on the other hand, this annotation doesn't really do anything. It is more like a comment that IDE's and such can recognize. Jason coincidently did the same in his draft PR: https://github.com/sonatype/package-url-java/pull/13, so unless Jason objects, I think we should leave it.

Reply:
yar, could be moved to javadoc too; its just meant as a hint to developer as to why the access/existence is as-so.  but at the end of the day I don't care too much

Reply:
ah, figured this out, it's because Guava defines `@VisibleForTesting`

That's not totally evident to outsider. I might've guessed this method used to be visible for just testing but now it's visible for other reasons (and I'd have been wrong)